src/ontology/mondo-edit.obo

```diff
@@ -463675,6 +463789,13 @@ def: "A cognitive disorder that is characterized by the presence of significant
 xref: DOID:0080831 {source="MONDO:equivalentTo"}
 is_a: MONDO:0002039 {source="DOID:0080831"} ! cognitive disorder
 
+[Term]
+id: MONDO:0850295
```

Comment:
can you pls add the logical def according to the acquired pattern?
thanks!

Reply:
added the subclassOf axiom and not the logical def because there is no term for "laryngomalacia"